assets/js/theme/common/carousel/utils/handleImageAspectRatio.js

```diff
@@ -17,12 +17,20 @@ const defineClass = (imageAspectRatio) => {
     }
 };
 
+const installClass = (imageNode, $slides) => {
```

Comment:
I believe setClass is more meaningful name here

Reply:
I can change it if you insist, but honestly I think it is the same